File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGWrap.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGWrap //Type originates from: YGEnums.h
{
    YGWrapNoWrap,
    YGWrapWrap,
    YGWrapWrapReverse;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGWrap forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGWrap.kt
package app.cash.redwood.yoga.internal.enums

/** Type originates from: YGEnums.h */
enum class YGWrap {
  YGWrapNoWrap,
  YGWrapWrap,
  YGWrapWrapReverse;

  fun getValue(): Int {
    return ordinal
  }

  companion object {
    fun forValue(value: Int): YGWrap {
      return values()[value]
    }
  }
}
